playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -115,6 +116,7 @@
   - role: openshift_sdn
     when: openshift_use_openshift_sdn | default(True) | bool
 
+
```

Comment:
Extra empty line

Reply:
@vrutkovs Thank you for the review comments. I have fixed it. 
Please let us know if we can proceed further.